zyte_common_items/base.py

```diff
@@ -48,7 +48,16 @@ def _extend_trail(trail: _Trail, key: Union[int, str]):
 
 
 @attrs.define
-class Item(_ItemBase):
+class ProbabilityMixin:
+    def get_probability(self) -> Optional[float]:
+        if metadata := getattr(self, "metadata", None):
+            if probability := getattr(metadata, "probability", None):
+                return probability
```

Comment:
Wouldn't it return None when the probability is 0?

Reply:
Good catch! https://github.com/zytedata/zyte-common-items/pull/65/commits/e5cececd92943c248d45c355768762713c5df220